browser/ai_chat/ai_chat_settings_helper.cc

```diff
@@ -201,6 +204,15 @@ void AIChatSettingsHelper::GetDefaultModelKey(
   std::move(callback).Run(model_service_->GetDefaultModelKey());
 }
 
+void AIChatSettingsHelper::DeleteConversations() {
+  ai_chat::AIChatService* service =
+      ai_chat::AIChatServiceFactory::GetForBrowserContext(profile_);
+  if (!service) {
+    return;
+  }
+  service->DeleteConversations();
```

Comment:
I wonder if we just pass the `AIChatService` mojo to the UI instead of passing it via the `AIChatSettingsHelper` mojo?

Reply:
I don't think we ever passed native services to java via mojo. Why do we need that?

Reply:
we are passing AIChatSettingsHelper to Java and its calling a function which then calls the service function only. So why not just pass the Service to Java now that we have one? AIChatSettingsHelper was made before AIChatService and now that it exists maybe we shouldn't add new things to AIChatSettingsHelper

Reply:
as per discussion in DM we decided to leave as is as there are questions about the service lifetime, there has to be a native object that hold a pointer for Java.